classes/lang/KeysReference/FeatureFlagLang.php

```diff
@@ -91,3 +91,7 @@
 // Order states feature flag
 trans('Order states', 'Admin.Advparameters.Feature');
 trans('Enable / Disable the migrated order states page.', 'Admin.Advparameters.Help');
+
+// Carts index feature flag
+trans('Carts', 'Admin.Advaparameters.Feature');
+trans('Enable / Disable the migrated carts index page.', 'Admin.Advparameters.Help');
```

Comment:
Oh, here aswell I think

```suggestion
trans('Enable / Disable the migrated carts page.', 'Admin.Advparameters.Help');
```

Reply:
I agree and feature_flag.xml as well!